docroot/modules/custom/mass_content/mass_content.module

```diff
@@ -1083,3 +1090,30 @@ function mass_content_crop_update(EntityInterface $entity) {
     }
   }
 }
+
+/**
+ * Sets the sort_bef_combine URL parameter to sort Collection views.
+ */
+function mass_content_set_default_sort_from_collection(Term $collection) {
+  if (!$collection) {
```

Comment:
Is this possible, given the `Term` typehint above? It looks like `mass_content_get_collection_from_current_page` can return `null`, so we either need to move this check to  `mass_content_views_pre_view` or add `= NULL` to the function signature.

Reply:
This is not related to my ticket, this came from the develop branch, when I merged today. Do you think I should change this in the scope of this ticket? @deviantintegral

Reply:
Leaving this as is, since there are several places where this function is used.